### ISSUE TITLE
emphasis on industry

### ISSUE THREAD

**Issue by FabienGandon** [w3c/AB-public#75]
In the [Vision for the W3C](https://github.com/w3c/AB-public/tree/main/Vision#vision-for-the-w3c) there is this sentence

> The W3C is an association where diverse voices from around the world and industries come together

While I agree with this emphasis on diversity I wonder about the emphasis on industry as opposed, for instance, to public sector actors.

**Comment by frivoal** [comment]
Would rephrasing this as "verticals" help?

**Comment by michaelchampion** [comment]
"Verticals" is a bit jargon-y for my taste. "Areas of expertise"? Maybe enumerate "diverse voices from industries, academia, interest groups, and governments around the world?"  Or just say "diverse voices" without mentioning any specific axes of diversity?

**Comment by FabienGandon** [comment]
IMHO, the enumeration emphasises the idea of diversity so that would be my preference.

**Comment by fantasai** [comment]
@michaelchampion I think we do need to enumerate axes, because otherwise “diversity” will be interpreted by many as meaning “from different demographic categories” only.

**Comment by frivoal** [comment]
> Industry , Academia, Government, Interest/Advocacy groups

Having people from each of those is a form of diversity, but "industries" or "verticals", as a plural, was trying to hint at something else. Car manufacturers, and payment providers, and browser vendors, and book publishers, and media distribution…

"fields of activity"?

> ... ?

Also: entity size: from governments and multi nationals to SMEs or individuals.

**Comment by TzviyaSiegman** [comment]
@michaelchampion and @fantasai  we have attempted to enumerate axes of diversity in the past, but it is not necessarily a productive task, Many people "check off" multiple boxes, and this can lead to a tokenist approach. Not all items are visible. 

CEPC lists the following as areas not to discriminate "socio-economic status, sexual orientation, religion, race, physical appearance, neurotype, nationality, mental health, language, indigeneity, immigration status, gender, gender identity, and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age", but this does not include issues like company size or skillset.

**Comment by michaelchampion** [comment]
I guess we should step back and ask "what are we trying to accomplish with this sentence?" Clearly affirming that we want to hear from "different voices" is critical, and the very notion of a "World Wide Web" implies we want to hear from people across  geographical/linguistic/cultural boundaries.  Beyond that maybe just reference the CEPC list?

**Comment by cwilso** [comment]
The purpose of this sentence was specifically calling out two critical axes of diverse voices:
- global diversity: geographical/cultural/linguistic
- industry diversity: companies and industries across a wide array of different industries (not just tech sector)

**Comment by frivoal** [comment]
I think keeping the word "industries" plural is key to what this is trying to say.

**Comment by agreiner** [comment]
I think the word "industry" is essentially plural, as it applies to all industry rather than just a specific one. See definition 1a, https://www.merriam-webster.com/dictionary/industry

**Comment by dwsinger** [comment]
I agree that we're trying to document diversity of reason-for-engagement, and I would agree with James' list but add something like "including" so that if we leave someone out, they don't feel excluded.

diverse stakeholders such as those from academia, government, industry, and civil society…

**Comment by mnot** [comment]
Saying "industry" or "industries" sends a strong message that the Web is primarily commercial.

Noting that there's a bullet point immediately following that explores diversity in more detail, removing "and industries" is appropriate -- leaving it in priviledges commercial interests and sends the wrong message. If we need to expand upon diversity more, the bullet point is the right place to do it.

**Comment by agreiner** [comment]
Yes, I am very much in agreement that we shouldn't say "industry" without mentioning other types of stakeholders in a way that gives them equal weight.

**Comment by frivoal** [comment]
> we shouldn't say "industry" without mentioning […]


I think we could do better than the current phrasing, but some of the proposals here seem to miss or dismiss what the current phrasing is trying to achieve.

Industry, in the singular, seems unhelpful to me. Yes, it encompasses all of industry, and so it is not exclusionary, but all it does is insist on the commercial nature. And indeed, as has been pointed out, that's not helpful, as there is more to the web than that.

Industries, in the plural, is trying to do something different. It's saying: Not just browsers; also publishing; also music; also cinema; also retail; also education; also healthcare; also video games; also banking; also tourism…

If we insist on having "industry and public sector actors", then it feels like if we have some commercial players (say browsers) and some government representatives, then we're doing a good job, because industry as well as public sector are covered. Industr**ies** points to the fact that no, it isn't good enough, and the private sector itself is hugely diverse, and that we're missing something important if we don't have breadth in that area.

Now, I do agree that this is not only about the private sector. But in my view, private sector, or industries, is only worth mentioning at all if it is to speak to the fact that it is diverse and that this diversity needs representation. I am not especially tied to the word "industry/ies" itself (as can be seem from my earlier ill fated suggestion to talk about verticals instead), but plurality is the whole point of that sentence.

> Noting that there's a bullet point immediately following that explores diversity in more detail

That bullet point focuses on aspects of diversity that relate to personal characteristics. That is very clearly important, and it absolutely needs to be stated, but this point about "industries" is trying to say that also need institutional diversity, and that individual diversity, while necessary, is not sufficient. I wouldn't muddle that bullet point, because individual diversity is important to highlight. Whether the other part needs to stay where it is, or become a new bullet point, I am more neutral about.

Now, if we could find a term that kept the plurality that "industries" is trying to point at, and expanded to also cover non-profit or public sector fields of activity, that would work for me. Actually, I'd be even happier if we could find a term or set of terms that also hinted types and sizes of organizations. Multinationals and small businesses and NGOs and government agencies and… , in all sorts of fields of actives.

**Comment by mnot** [comment]
@frivoal you might be looking for “multi-stakeholder”.

**Comment by mnot** [comment]
See also https://github.com/WebStandardsFuture/Vision/issues/28 which is #10 here.

**Comment by FabienGandon** [comment]
reviewing the latest version (BTW thanks a lot for this work) I realise that al equivalent sentence appears in the `Mission`

`(...) where diverse voices from around the world and from different industries work together (...)`

my suggestion in both cases would be something like

`(...) where diverse voices from around the world and from different organizations and industries work together (...)`

opening the participation to W3C to more than enterprises that produce or supply goods, services, or sources of income.

**Comment by avneeshsingh** [comment]
This statement was written from high level point of view.
It is obvious that world wide web needs voices from around the world.
The "industries" was included to emphasize various applications of web in different industries.
If we achieve world wide participation, from North America, South America, Europe, Africa, Asia, Australia, but we end up having only one industry in W3C for example publishing industry, then it defeats the purpose. We need to ensure that different industries are represented, browsers, publishing, automotive, payments and so on.

To clarify it further: If a university creates a web browser, in my understanding it should be considered as a part of web industry, though it is created under a university.

Some additional points:
- We should make sure that words used by us are understood by the worldwide community. Terms like "verticals" look quite specialized due to this reason.
- Due to same reason the long list of terms would also not be desirable.

**Comment by dwsinger** [comment]
I wonder if 'sectors' would be a way of expressing that we want varied areas of interest including diverse commercial areas (which is what 'industries' suggests a restriction to)?

**Comment by frivoal** [comment]
It's not obvious to me that the suggestion from @FabienGandon at https://github.com/w3c/AB-public/issues/75#issuecomment-1578772035 solves the issue, but I don't find it objectionable either, so if enough people like it, I could live with it. That said, @avneeshsingh has suggested flipping the order ("different organizations and industries" -> "different industries and organizations"), and I agree that would be better.

As to @dwsinger 's https://github.com/w3c/AB-public/issues/75#issuecomment-1582658568, maybe "different industries and sectors" (as it "public sector", "non-profit sector", "private sector"…) would work?

**Comment by fantasai** [comment]
The most recent rewrite has indeed flipped the order as @avneeshsingh suggested, and I think it does seem to address the issue.
